application/config/config-defaults.php

```diff
@@ -80,6 +80,8 @@
 $config['allowexportalldb']          = 0; // Default 0 will only export prefixed tables when doing a database dump. If set to 1 ALL tables in the database will be exported (use carefully)
 $config['maxDatabaseSizeForDump']    = 256; // Maximum database size in megabytes to be able to download without errors
 
+$config['non_numerical_answer_prefix'] = 'A'; // Non-Numerical answer option prefix.
```

Comment:
please rename it to 

answer_code_prefix
and
subquestion_code_prefix

In the setting comment it needs to be clear that it has to start with a character.